input_data.json

```
@@ -1,15 +1,34 @@
 {
-    "minimum_x": -10.0,
+    "minimum_x": 0.0,
     "maximum_x": 10.0,
-    "number_of_voxels": 7,
+    "positions": {
+        "0": [
```

Can you use integer keys instead of strings?
```suggestion
        0 : [
```

(and then avoid some of my other comments about converting these to integers)